### PR TITLE
Accuracy aware training documentation

### PR DESCRIPTION
**Pull request by vanyalzr**

### CODE REVIEW

File: docs/Usage.md [hunk 1]
```diff
@@ -158,3 +158,86 @@ class MyModule(torch.nn.Module):
 If registered module should be ignored by specific algorithms use `ignored_algorithms` parameter of decorator.
 
 In the example above, the NNCF-compressed models that contain instances of `MyModule` will have the corresponding modules extended with functionality that will allow NNCF to quantize, sparsify or prune the `weight` parameter of `MyModule` before it takes part in `MyModule`'s `forward` calculation.
+
+### Accuracy-aware model training
```
Comment (AlexKoff88) [inline docs/Usage.md:162]: I suggest adding the information about AA into the Key Features list in the main README with a reference to this doc.

File: docs/Usage.md [hunk 2]
```diff
@@ -158,3 +158,86 @@ class MyModule(torch.nn.Module):
...
+NNCF has the capability to execute an internal custom compression-aware training loop in order to automate away some of the manual hyperparameter search related to model training such as setting the total number of epochs, the target compression rate for the model, etc. The currently supported training loop (encapsulated in the `AdaptiveCompressionTrainingLoop` class) is targeted for the automated discovery of the compression rate for the model given that it satisfies the user-specified maximal tolerable accuracy drop due to compression. The training loop could be run with either PyTorch or TensorFlow backend with the same user interface of the training loop (except for the TF case where the Keras API is used for training).
```
Comment (AlexKoff88) [inline docs/Usage.md:163]: @vanyalzr, can you please simplify the first sentence and start from general words, like "NNCF has the capability to apply optimization algorithms while satisfying given accuracy constraints...."

File: docs/Usage.md (hunk 1 above)
Comment (vanyalzr) [reply docs/Usage.md:162]: Done

File: docs/Usage.md (hunk 2 above)
Comment (vanyalzr) [reply docs/Usage.md]: Done

**Review by vshampor — APPROVED** [review]

**Review by AlexKoff88 — APPROVED** [review]
